### PR TITLE
Compile safe lookups to individual methods

### PR DESCRIPTION
**Pull request by vctrshn**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @vshen94 has signed the CLA. Thanks for the pull request!

File: build/parsers/Parser.y [hunk 1]
```diff
@@ -82,14 +82,23 @@
...
+     * @see \Box\Brainy\Brainy::$safe_lookups
      */
     public function compileVariable($variable)
     {
         $unsafe = '$_smarty_tpl->tpl_vars[' . $variable . ']';
-        if ($this->safe_lookups === 0) { // Unsafe lookups
-            return $unsafe;
+
+        switch ($this->safe_lookups) {
+            case 0: //LOOKUP_UNSAFE
```
Comment (mattbasta) [inline build/parsers/Parser.y]: I'd just reference the constant, on each of these.

```
case \Box\Brainy\Brainy::LOOKUP_UNSAFE:
```

Comment (vctrshn) [inline build/parsers/Parser.y]: done

**Comment by mattbasta** [comment]
Looks great! r+

**Pull request merged**